cards/time-lord.md

```diff
@@ -4,8 +4,8 @@
 ### Time of Play: Any
 
 ### Card Intent & Rules:
-The player who made the play resulting in this card being drawn is given the title of **_Time Lord_**. The **_Time Lord_** has the ability to roll back works like the _Time Warp_ expect is indefinite. If the **_Time Lord_** is the server when his/her team loses he can continue serving and roll back the last point until his team scores. This card is indefinite throughout the game so each time the **_Time Lord_** rotates in to serve this power is active.
+The player who made the play resulting in this card being drawn is given the title of **_Time Lord_**. The **_Time Lord_** has the ability to roll back works like the _Time Warp_ except it is indefinite. If the **_Time Lord_** is the server when when their team loses they can continue serving and roll back the last point until their team scores. This card is indefinite throughout the game so each time the **_Time Lord_** rotates in as the server this power is active.
 
 ### Card Text:
 **Time Lord**
-The player who caused the draw becomes the Time Lord for the remainder of the game. If the Time Lord is serving and the opposing team scores the winning point the Time Lord can replay the point until his/her team scores.
\ No newline at end of file
+The player who caused the draw becomes the Time Lord for the remainder of the game. If the Time Lord is serving and the opposing team scores the winning point the Time Lord can replay the point until thier team scores.
```

Comment:
"...point until thier team scores." should become "...point until their team scores."